commons/marmotta-sesame-tools/marmotta-model-vocabs/pom.xml

```diff
@@ -26,7 +26,7 @@
     </parent>
 
     <artifactId>marmotta-model-vocabs</artifactId>
-    <packaging>jar</packaging>
+    <packaging>bundle</packaging>
```

Comment:
why `marmotta-model-vocabs` here? it has nothing to do with either LDPath or LDCache...